src/Components/Alert/Components/NotificationPreferences.tsx

```diff
@@ -39,14 +39,6 @@ export const NotificationPreferences: FC<NotificationPreferencesProps> = ({
     }
   )
 
-  useEffect(() => {
```

Comment:
Removing this effect is enough because the default is 'true' and here it was set to false in case the user had unsubscribed from these emails.